File: HomeWorks/src/Lesson6/Hands/ToshibaHand.java
package Lesson6.Hands;

public class ToshibaHand implements IHand {
    private int price;

    public ToshibaHand() {}

    public ToshibaHand (int price){
        this.price = price;
    }

    @Override
    public void upHand() {
        System.out.println("Машет рука Toshiba");
    }

    public void setPrice(int price) {
        this.price = price;
    }

    @Override
    public int getPrice() {
        return price;
    }
}

File: HomeWorkKotlin/src/Robot/Hands/ToshibaHand.kt
package Robot.Hands

class ToshibaHand(private val priceParam : Int) : IHand {

    override var price = priceParam
        set(value) {
            if(value > 0) field = value
        }

    override fun upHand() {
        println("Машет рука Toshiba")
    }
}